Jenkinsfile

```diff
@@ -490,6 +506,107 @@ az aks create                      \
             }
         }
     }
+*/
+
+    //def eksVersions = ["1.10", "1.11"]
+    def eksVersions = ["1.11"]
+    for (x in eksVersions) {
+        def kversion = x // local bind required because closures
+        def platform = "eks-${kversion}"
+        platforms[platform] = {
+            stage(platform) {
+                node(label) {
+                    withCredentials([[
+                        $class: 'AmazonWebServicesCredentialsBinding',
+                        credentialsId: 'aws-eks-kubeprod-jenkins',
+                        accessKeyVariable: 'AWS_ACCESS_KEY_ID',
+                        secretKeyVariable: 'AWS_SECRET_ACCESS_KEY',
+                    ]]) {
+                        withEnv([
+                            "AWS_DEFAULT_REGION=eu-central-1",
```

Comment:
FWIW, you can add this as an environment variable for the aws-cli container in the PodTemplate

Reply:
Also: we should run the tests on the same side of the planet as jenkins, or all requests to/from the cluster are unnecessarily slow :stuck_out_tongue: 
Best would be us-east-something.

Reply:
Done